packages/core/helper-plugin/package.json

```diff
@@ -71,10 +71,23 @@
     "browserslist-to-esbuild": "1.2.0",
     "cross-env": "^7.0.3",
     "esbuild-loader": "^2.21.0",
+    "match-sorter": "^4.0.2",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "5.3.4",
     "react-test-renderer": "^17.0.2",
-    "require-from-string": "2.0.2",
     "rimraf": "3.0.2",
-    "typescript": "4.6.2"
+    "styled-components": "5.3.3",
+    "typescript": "4.6.2",
+    "webpack": "^5.75.0"
+  },
+  "peerDependencies": {
+    "@strapi/design-system": "1.6.3",
+    "@strapi/icons": "1.6.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "5.3.4",
+    "styled-components": "5.3.3"
   },
```

Comment:
this is a list of deps that we:
* need to run the library
* do not want to install a "localised" version for the plugin to use

So instead it will use the projects dependencies. This solves a lot of issues with "can't find X"

Reply:
Hence why they're also in the `devDeps`